.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,80 @@
+name: Bug Report
+description: File a report for unexpected behavior in SuperTux.
+title: "[Bug]: "
+labels: ["type:bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    id: supertux-version
+    attributes:
+      label: SuperTux Version
+      description: The version of SuperTux, shown at the bottom-left corner on the title screen.
+      placeholder: ex. v0.6.3-1236-g7451121dd
+    validations:
+      required: true
+  - type: input
+    id: system-info
+    attributes:
+      label: System Information
+      description: Details about your operating system (such as Linux distribution or Windows version) and CPU architecture (ex. 64-bit or 32-bit).
+      placeholder: ex. Windows 11 64-bit
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected Behavior
+      description: What did you expect to happen?
+      placeholder: Describe the behavior you expected in detail.
+    validations:
+      required: true
+  - type: textarea
+    id: actual-behavior
+    attributes:
+      label: Actual Behavior
+      description: Instead, what actually happened?
+      placeholder: Describe the actual behavior in detail.
+    validations:
+      required: true
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: Steps To Reproduce Actual Behavior
+      description: A clear and concise description of how the actual behavior was achieved, since starting up the game.
+      placeholder: |
+        Describe the steps, leading to the actual behavior, preferably with a numerated list.
```

Comment:
Not sure about the first comma here.

Reply:
https://readingwriting.byuh.edu/reading-writing-center/microlab-3-comma-usage at:

> COMMAS THAT ENCLOSE

> VII. Nonrestrictive phrases:
>
> Use commas to enclose groups of words that provide additional information not essential to the basic meaning of the sentence.

> VIII. Restrictive phrases:
>
> A group of words that provides descriptive or identifying information essential to the basic meaning of the sentence > should not be enclosed by commas and is called a restrictive phrase.

There is both restrictive and nonrestrictive phrases. So if "leading to the actual behavior" is something deemed as essential to the meaning of the sentence, it's a restrictive phrase, meaning you're right. I kind of think it actually is, but I'm just not too sure about that either.